tools/cli/persistenceUtil.go

```diff
@@ -83,19 +95,32 @@ func CreateDefaultDBConfig(c *cli.Context) (config.DataStore, error) {
 		}
 
 		if c.Bool(FlagEnableTLS) {
-			defaultConfig.TLS = &auth.TLS{
-				Enabled:                true,
-				CertFile:               c.String(FlagTLSCertPath),
-				KeyFile:                c.String(FlagTLSKeyPath),
-				CaFile:                 c.String(FlagTLSCaPath),
-				EnableHostVerification: c.Bool(FlagTLSEnableHostVerification),
-			}
+			defaultConfig.TLS = tls
 		}
 
 		defaultStore := config.DataStore{
 			Cassandra: defaultConfig,
 		}
 
+		return defaultStore, nil
+	} else if engine == mySQLDBType {
+		addr := fmt.Sprintf("%v:%v", c.String(FlagDBAddress), c.Int(FlagDBPort))
+		defaultConfig := &config.SQL{
+			User:         c.String(FlagUsername),
+			Password:     c.String(FlagPassword),
+			DatabaseName: c.String(FlagKeyspace),
+			ConnectAddr:  addr,
+			PluginName:   engine,
+		}
+
+		if c.Bool(FlagEnableTLS) {
```

Comment:
same here, you can just remove the if check and assign it when initializing defaultConfig